SYCL/InlineAsm/include/asmhelper.h

```diff
@@ -123,17 +119,29 @@ bool launchInlineASMTestImpl(F &f, bool requires_particular_sg_size = true) {
 /// \returns false if test wasn't launched (i.e.was skipped) and true otherwise
 template <typename F>
 bool launchInlineASMTest(F &f, bool requires_particular_sg_size = true,
-                         bool need_to_throw_exception = false) {
+                         bool need_to_throw_exception = false,
```

Comment:
Since you moved check for exception message into the helper, I suggest to get rid of `need_to_throw_exception` argument completely, as we don't have tests that expect exceptions out of this function